template_feed/Microsoft.DotNet.Common.ItemTemplates/content/BuildProps/Directory.Build.props

```diff
@@ -6,5 +6,9 @@
 <!--#endif-->
   <PropertyGroup>
 
+<!--#if useArtifacts -->
+  <ArtifactsPath>$(MSBuildThisFileDirectory)artifacts</ArtifactsPath>
```

Comment:
@dsplaisted to confirm this is the correct expression. I think it is, but safety-first!